src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/PossibleDeNovo.java

```diff
@@ -38,48 +40,75 @@
  *
  */
 @DocumentedFeature(groupName=HelpConstants.DOC_CAT_ANNOTATORS, groupSummary=HelpConstants.DOC_CAT_ANNOTATORS_SUMMARY, summary="Existence of a de novo mutation in at least one of the given families (hiConfDeNovo, loConfDeNovo)")
-public final class PossibleDeNovo extends InfoFieldAnnotation {
+public final class PossibleDeNovo extends PedigreeAnnotation {
+    protected final Logger warning = LogManager.getLogger(this.getClass());
+    private final MendelianViolation mendelianViolation;
+    private Set<Trio> trios;
 
+    @VisibleForTesting
     public PossibleDeNovo(final Set<Trio> trios, final double minGenotypeQualityP) {
+        super((Set<String>) null);
         this.trios = Collections.unmodifiableSet(new LinkedHashSet<>(trios));
         mendelianViolation = new MendelianViolation(minGenotypeQualityP);
     }
 
-    /**
-     * This is dummy constructor that will do nothing until https://github.com/broadinstitute/gatk/issues/1880 is addressed
-     */
-    public PossibleDeNovo(){
-        this(Collections.emptySet(), 0);
+    public PossibleDeNovo(final File pedigreeFile){
+        super(pedigreeFile);
+        mendelianViolation = new MendelianViolation(DEFAULT_MIN_GENOTYPE_QUALITY_P);
     }
 
-    protected final OneShotLogger warning = new OneShotLogger(this.getClass());
+    public PossibleDeNovo(){
+        this( null);
```

Comment:
Tests are failing now because this is passing a null pedigree file to a method that doesn't check for null. These really should all have no-arg constructors, but as a hack to get this working for now, it should probably has to be:
```suggestion
        super((Set<String>) null);
        mendelianViolation = new MendelianViolation(DEFAULT_MIN_GENOTYPE_QUALITY_P);
```